src/pykiso/tool/pykiso_to_pytest/cli.py

```diff
@@ -117,7 +117,8 @@ def main(filename: str, destination: str):
     pykiso_config = config_parser.parse_config(filename)
 
     template_loader = FileSystemLoader(searchpath=str(ROOT_PATH / "templates"))
-    template_env = Environment(loader=template_loader)
+    # autoexcape needs to be switched off, since executable python code is included.
+    template_env = Environment(loader=template_loader, autoescape=False)  # nosec B701
```

Comment:
Why an ignore again?

Reply:
I think you need to extend the comment to add a little bit more context regarding why this error is not relevant for us. This needs to be done to all error suppressed. As code reader, I should not have the feeling it is suppressed because I do not want to deal with the error. I need to see that the warning was understood and I explicitly ignore it. What would be also interesting is to exercise the mind by thinking, what changes would be needed to not have this error? Any design? any refactoring? Maybe check with @BKaDamien, @cody82 and @sebclrsn.